src/utils.js

```diff
@@ -70,3 +71,22 @@ export function pauseEvent(e) {
   e.stopPropagation();
   e.preventDefault();
 }
+
+export function getKeyboardValueMutator(e) {
+  switch (e.keyCode) {
+    case keyCode.UP:
+    case keyCode.RIGHT:
+      return (value, props) => value + props.step;
+
+    case keyCode.DOWN:
+    case keyCode.LEFT:
+      return (value, props) => value - props.step;
+
+    case keyCode.END: return (value, props) => props.max;
+    case keyCode.HOME: return (value, props) => props.min;
+    case keyCode.PAGE_UP: return (value, props) => value + props.step * 2;
```

Comment:
> For the up and down behaviours I tend to follow the Polymer approach. Mainly because you are "upping" or "downing" the current value.

What I tried to explain was that I like how Polymer's approach had the same type of behaviour with both `up` and `pageup` keys. Both "up" keys increased the current value. OAA's behaviour was the same for `up` and `pagedown`, which I didn't like because of mixed "up" and "down" keys.

Later on I tried to reason that OAA has a point about the increased value jump through `pageup` and `pagedown`. For small sliders it makes no different, but if you have a big one you would probably want to go faster through the steps. 

> OAA Accessibility has a point about the page up and page down behaviours. I mean, left/down and up/right only go by 1 step. For small sliders this is fine, but for large ones not. Also if an user actually uses the page up/page down key, they probably expect something more then just a mimick of left and right.

I'm open for both approaches. Either increase the step when using the page keys or just copy the same behaviour from the normal up/down keys. It's up to you guys.